src/LinAlg/hiopLinSolverIndefDenseMagma.cpp

```diff
@@ -119,9 +136,40 @@ namespace hiop
     }
     //printf("(pos,null,neg)=(%d,%d,%d)\n", posEigVal, nullEigVal, negEigVal);
     nlp_->runStats.linsolv.tmInertiaComp.stop();
-    
-    if(nullEigVal>0) return -1;
-    return negEigVal;
+
+    return true;
+  }
+
+  bool hiopLinSolverIndefDenseMagmaBuKaDev::compute_inertia(double **MM, 
```

Comment:
```suggestion
  bool hiopLinSolverIndefDenseMagmaBuKaDev::compute_inertia(double * MM, 
```
Perhaps it would be better if we pass the pointer to matrix data block here? That would avoid dereferencing pointer a few lines below.